tide_landing_page.install

```diff
@@ -880,3 +880,164 @@ function tide_landing_page_update_8021() {
+  $config->save();
+}
+
+/**
+ * Imports new fields to introduction_banner bundle.
+ */
+function tide_landing_page_update_8024() {
+  $configs = [
+    'field.storage.paragraph.field_call_to_action_title' => 'field_storage_config',
+    'field.storage.paragraph.field_banner_type' => 'field_storage_config',
+    'field.storage.paragraph.field_banner_display_type' => 'field_storage_config',
+    'field.field.paragraph.introduction_banner.field_paragraph_body' => 'field_config',
+    'field.field.paragraph.introduction_banner.field_call_to_action_title' => 'field_config',
+    'field.field.paragraph.introduction_banner.field_banner_type' => 'field_config',
+    'field.field.paragraph.introduction_banner.field_banner_display_type' => 'field_config',
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $config_entity = $storage->createFromStorageRecord($config_read);
+    $config_entity->save();
+  }
+
+  // Changes Feature links to Links.
+  $field_paragraph_links = \Drupal::configFactory()
+    ->getEditable('field.field.paragraph.introduction_banner.field_paragraph_links');
+  $field_paragraph_links->set('label', 'Links');
+  $field_paragraph_links->save();
+}
+
+/**
+ * Migreates field_paragraph_summary to field_paragraph_body.
+ */
+function tide_landing_page_update_8025(&$sandbox) {
+  $wysiwyg = 'field_paragraph_body';
+  if (!isset($sandbox['total'])) {
+    $count = \Drupal::entityTypeManager()
+      ->getStorage('paragraph')
+      ->getQuery()
+      ->condition('type', 'introduction_banner')
+      ->count()
+      ->execute();
+    $sandbox['total'] = $count;
+    $sandbox['current'] = 0;
+    $sandbox['processed'] = 0;
+    $sandbox['#finished'] = $count ? 0 : 1;
+  }
+  $batch_size = 10;
+  $paragraph_ids = \Drupal::entityTypeManager()
+    ->getStorage('paragraph')
+    ->getQuery()
+    ->condition('id', $sandbox['current'], '>')
+    ->condition('type', 'introduction_banner')
+    ->sort('id', 'ASC')
+    ->range(0, $batch_size)
+    ->execute();
+  foreach ($paragraph_ids as $paragraph_id) {
+    $sandbox['current'] = $paragraph_id;
+    $paragraph = Paragraph::load($paragraph_id);
+    if ($paragraph && $paragraph->hasField($wysiwyg) && $paragraph->hasField('field_paragraph_summary')) {
+      $paragraph->{$wysiwyg}->value = $paragraph->field_paragraph_summary->value;
+      $paragraph->{$wysiwyg}->format = 'rich_text';
+      $paragraph->setNewRevision(FALSE);
+      $paragraph->save();
+    }
+    $sandbox['processed']++;
+  }
+  $sandbox['#finished'] = ($sandbox['processed'] / $sandbox['total']);
```

Comment:
Hi @vincent-gao I am getting an error on this line in content-together-vic:

```
Division by zero tide_landing_page.install:1017                        [warning]
Performing tide_landing_page_update_8025                                    [ok]

PHP Fatal error:  Allowed memory size of 419430400 bytes exhausted (tried to allocate 38540224 bytes) in /app/vendor/drush/drush/includes/backend.inc on line 180
```

It doesnt performs any other updates due to the error.

Reply:
Hi @asarfara , weird , I will take a look.

Reply:
Hey @asarfara please see https://github.com/dpc-sdp/tide_landing_page/pull/104